Sharpcaster/Models/Media/Item.cs

```diff
@@ -24,6 +24,16 @@ public class Item
         /// Gets or sets a value indicating whether autoplay is enabled or not
         /// </summary>
         [DataMember(Name = "autoplay")]
-        public bool Autoplay { get; set; }
+        public bool Autoplay { get; set; } = true;
+
+
+        [DataMember(Name = "orderId")]
+        public long OrderId { get; set; }
+
+        [DataMember(Name = "startTime")]
+        public long StartTime { get; set; }
+
+        //[DataMember(Name = "preloadTime")]
```

Comment:
can you add these here:
        [DataMember(Name = "preloadTime")]
        public long PreloadTime { get; set; }

        [DataMember(Name = "activeTrackIds")]
        public long[] ActiveTrackIds { get; set; }